dockstore-webservice/src/main/java/io/dockstore/webservice/languages/LanguagePluginHandler.java

```diff
@@ -72,6 +73,29 @@ public Version parseWorkflowContent(String filepath, String content, Set<SourceF
             version.addAuthor(author);
         }
         version.setDescriptorTypeVersionsFromSourceFiles(sourceFiles);
+        if (version.getVersionMetadata().getDescriptorTypeVersions().isEmpty()) {
```

Comment:
We don't have this check for the other languages; we always recalculate the versions. I lean towards recalculating in case there's ever a bug where we need to run it again, but don't feel super strongly. But we should be consistent whichever way.